config/application.config.yml

```diff
@@ -0,0 +1,2 @@
+GOOGLE_APP_ID:
+GOOGLE_APP_SECRET:
```

Comment:
no, that is so other developers know what they need in the actual `application.yml` file without housing the actual values.